sycl/test-e2e/Matrix/get_coord_int8_matB.cpp

```diff
@@ -9,15 +9,9 @@
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
-// XFAIL: cpu
```

Comment:
I believe, we need to keep XFAIL: cpu, until this test starts to pass with OpenCL CPU runtime used with intel/llvm.